modules/shortcodes/js/blocks/vr-block.jsx

```diff
@@ -0,0 +1,94 @@
+/* global wp */
+/* eslint react/react-in-jsx-scope: 0 */
+
+( function( blocks, components, i18n ) {
+	const {
+		registerBlockType,
+		UrlInput
+	} = blocks;
+	const {
+		Placeholder,
+		SelectControl
+	} = components;
+	const { __ } = i18n;
+
+	registerBlockType( 'gutenpack/vr', {
+		title: __( 'VR Image' ),
+		icon: 'embed-photo',
+		category: 'embed',
+		supportHTML: false,
```

Comment:
`supportHTML` is no longer supported.

```js
supports: {
    html: false,
},
```
is the way to go, see here: https://github.com/WordPress/gutenberg/blob/master/core-blocks/more/index.js#L34.

Reply:
Fixed!